app/workers/kiqs/process_promotional_tweet_worker.rb

```diff
@@ -0,0 +1,9 @@
+class ProcessPromotionalTweetWorker
+  include Sidekiq::Worker
+
+  def perform(promotional_tweet)
```

Comment:
You can't pass full objects in `Sidekiq` workers. You can pass it as YAML but that's not highly recommended. Can we pass it in as a hash of attributes to create the promotional_tweet?